decidim-core/lib/decidim/core.rb

```diff
@@ -591,6 +591,12 @@ def self.reset_all_column_information
     {}
   end
 
+  # The expiration time of the jwt tokens, after which issued token to the
+  # user will expired.
+  config_accessor :jwt_expiration_time do
+    10.minutes.to_i
+  end
```

Comment:
I believe this configuration belongs to the `Api` module instead of the core module. The Devise configuration block should be applied only if the `Api` module is installed.

Also, I think the previous default (1 hour) was OK for this config. Is there a reason it was reduced?

This is also still not configurable through environment variables, as per the prior review.